person.rb

```diff
@@ -1,15 +1,17 @@
 require_relative 'nameable'
 class Person < Nameable
+  attr_reader :rentals, :id
+
   def initialize(age, name = 'unkown', parent_permission: true)
     super()
     @id = Random.rand(1..1000)
     @name = name
     @parent_permission = parent_permission
     @age = age
+    @rentals = []
```

Comment:
- [ ] kindly ensure a  method for adding rentals should be present`
- [ ] also please the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association`